### DISCUSSION TITLE
Zlib error

### DISCUSSION THREAD

**Discussion by FaustTheKoR** [Ylianst/MeshCentral#4286] [Q&A] [answered]
Hi,

since yesterday, after 2 days off, when we launch Meshcentral on explorer we have : 

"ZLIB is not defined.  SRC zlib.js before zlib-crc32.js" error

Ubuntu 20.04 and mesh 1.49

Sincerely

**Comment by si458** [comment, marked as answer]
what version of nodejs are you using? (please make sure its at least LTS (16) as the has been a few changes)
also try updating to the latest meshcentral 1.0.54

**Reply by DaanSelen** [reply]
I still have this issue just after I added custom images to meshcentral through meshcentral-web.

**Comment by FaustTheKoR** [comment]
node version 12 - updated
meshcentral - updated

No more problem

Thanx :)
